root/collection/execTClonesArrayAbsorb.C

```diff
@@ -1,5 +1,5 @@
-{
-#ifdef ClingWorkAroundMissingImplicitAuto
+void execTClonesArrayAbsorb() {
+#ifndef ClingWorkAroundMissingImplicitAuto
```

Comment:
The reversion seems wrong.  The declaration needs to be present when the user request the workaround 'missing implicit auto keyword'.

Reply:
on the other hand there is no 'implicit auto' unless we have an unnamed macro.

Reply:
so the question really is, "is implicit now completely working?" (i.e. should we remove the #ifdef).